src/toil/provisioners/gceProvisioner.py

```diff
@@ -852,12 +876,9 @@ def _waitForAppliance(cls, ip_address, role, keyName='core'):
                     logger.info('...Still waiting for appliance, trying again in %s sec...' % sleepTime)
                     time.sleep(sleepTime)
             except:
```

Comment:
Now it will just eat up exception and keep retrying which is _not_ good behavior. What exception do we want to retry on?

Reply:
At this point, the connection works and docker is running. I don't know of any specific errors other than transient connection failures. What do you suggest?

Reply:
As far as I can tell, the only exception we would expect to encounter is from `_coreSSH` and is a `RuntimeError` so I think you should only retry on that exception or maybe only that exception and with a particular error code.